.gitignore

```diff
@@ -14,6 +14,7 @@
 # MemoryCaptures can get excessive in size.
 # They also could contain extremely sensitive data
 /[Mm]emoryCaptures/
+/[Ww]ebGL Builds/
```

Comment:
What you got in here?

Reply:
It's just the build files I got yesterday

Reply:
Can walk you through how to sort build and run it on WebGL at some point

Reply:
Ahh make sense, ya maybe when we get closer to the end